packages/tizen_rpc_port/lib/src/parcel.dart

```diff
@@ -0,0 +1,406 @@
+import 'dart:ffi';
+import 'dart:typed_data';
+import 'package:ffi/ffi.dart';
+
+import 'package:tizen_bundle/tizen_bundle.dart';
+import 'package:tizen_interop/6.5/tizen.dart';
+
+class _Timespec extends Struct {
+  @Int64()
+  external int tvSec;
+
+  @Int64()
+  external int tvNsec;
+}
+
+/// The timestamp when parcel created.
+class Timestamp {
+  Timestamp._(this.second, this.nanoSecond);
+
+  /// The second of timestamp.
+  int second;
+
+  /// THe nano second of timestamp. (10^-9 second)
+  int nanoSecond;
+}
+
+/// The header of parcel that is included information of the parcel.
+class ParcelHeader {
+  ParcelHeader._fromHandle(this._header);
+
+  late final Pointer<Void> _header;
+
+  /// The tag is used version naming from TIDL.
+
+  /// Gets the tag of header.
+  String get tag {
+    return using((Arena arena) {
+      final Pointer<Pointer<Char>> rawTag = arena();
+      final int ret = tizen.rpc_port_parcel_header_get_tag(_header, rawTag);
+      arena.using(rawTag.value, malloc.free);
+      if (ret != 0) {
+        throw ret;
+      }
+
+      return rawTag.value.toDartString();
+    });
+  }
+
+  /// Sets the tag of header.
+  set tag(String tag_) {
```

Comment:
We don't use trailing `_` in Dart.

```suggestion
  /// Sets a tag to this header.
  set tag(String value) {
```